libathemecore/version.c

```diff
@@ -1,82 +1,98 @@
-  "WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE",
-  "DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR",
-  "ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES",
-  "(INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;",
-  "LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON",
-  "ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT",
-  "(INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS",
-  "SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.",
-  " ",
-  "Currently Atheme's core group consists of the following developers,",
-  "in nick-alphabetical order:",
-  "  ilbelkyr, Janik Kleinhoff <janik@kleinhoff.de>",
-  "  mt, Max Teufel <max@teufelsnetz.com>",
-  "  siniStar, Austin Ellis <siniStar@IRC4Fun.net>",
-  " ",
-  "The following people have contributed blood, sweat and tears to",
-  "Atheme:",
-  "  elly, Elly <elly@leptoquark.net>",
-  "  jdhore, JD Horelick <jdhore1@gmail.com>",
-  "  jilles, Jilles Tjoelker <jilles@stack.nl>",
-  "  kaniini, William Pitcock <nenolod@nenolod.net>",
-  "  spb, Stephen Bennett <spb@exherbo.org>",
-  " ",
-  "  Adam, Adam <adam@sigterm.info>",
-  "  aji, Alex Iadicicco <alex@ajitek.net>",
-  "  alambert, Alex Lambert <alambert@quickfire.org>",
-  "  amdj, Aaron M. D. Jones <aaronmdjones@gmail.com>",
-  "  Attila, Attila Molnar <attilamolnar@hush.com>",
-  "  beu, Elfyn McBratney <elfyn.mcbratney@gmail.com>",
-  "  binki, Nathan Phillip Brink <ohnobinki@ohnopublishing.net>",
-  "  Dianora, Diane Bruce <db@db.net>",
-  "  danieldg, Daniel De Graaf <danieldegraaf@gmail.com>",
-  "  dwfreed, Douglas Freed <dwfreed@mtu.edu>",
-  "  gxti, Michael Tharp <gxti@partiallystapled.com>",
-  "  kog, Greg Feigenson <kog@epiphanic.org>",
-  "  kuja, Jeff Katz <jeff@katzonline.net>",
-  "  lu_zero, Luca Barbato <lu_zero@gentoo.org>",
-  "  marienz, Marien Zwart <marien.zwart@gmail.com>",
-  "  pfish, Patrick Fish <pofish@gmail.com>",
-  "  pippijn, Pippijn van Steenhoven <pippijn@one09.net>",
-  "  terminal, Theo Julienne <admin@ozweb.nu>",
-  "  tomaw, Tom Wesley <tom@tomaw.net>",
-  "  Trystan, Trystan Scott Lee <trystan@nomadirc.net>",
-  "  w00t, Robin Burchell <viroteck@viroteck.net>",
-  "  xiphias, Michael Rodriguez <dkingston@dkingston.net>",
-  "  zparta, Jens Holmqvist <zparta@hispan.se>",
-  " ",
-  "Portions of Atheme's code were derived from Shrike, which was mostly developed by",
-  "the following developers in nick-alphabetical order:",
-  "  madragoran, Mike Campbell <madragoran@avendesora.net>",
-  "  nenolod, William Pitcock <nenolod@nenolod.net>",
-  "  rakaur, Eric Will <rakaur@malkier.net>",
-  "  rintaun, Matt Lanigan <rintaun@projectxero.net>",
-  "  sycobuny, Stephen Belcher <sycobuny@malkier.net>",
-  " ",
-  "Visit our website at http://atheme.github.io",
-  0,
+	" ",
+	"Atheme IRC Services --",
+
```

Comment:
Hm, there's a blank line here that won't be in the output; everywhere else in the text, blank lines in the source only appear where there will be blank lines in the output as well. Just a minor nitpick, though.

Reply:
Fixed in commit 7bab1e5e718faa9bc237c493fdb7cb2f652b5492